robofile-templates/LaravelRoboFileTemplate.php

```diff
@@ -94,4 +147,18 @@ public function setupDev()
     {
         $this->_exec('php artisan db:seed --class=DatabaseSeeder');
     }
+
+    /**
+     * Ability to start, stop and restart imi queue service if it exists.
+     *
+     * @param $command
+     */
+    private function controlQueueServiceIfExisting($command)
```

Comment:
Maybe it would be better to put this in a separate task?

Reply:
Both is okay